wca-guidelines.md

```diff
@@ -51,6 +51,8 @@ To be more informative, each Guideline is classified using one of the following
 - 2c++) [ADDITION] Competitors must not provide intentionally misleading information, and returning competitors should provide information consistent with past information (e.g. exact name and WCA ID).
 - 2d+) [ADDITION] Date of birth and contact information should be especially secured.
 - 2d++) [RECOMMENDATION] If a third party (e.g. journalist) asks the organization team to be put in contact with any competitor(s), the competitor(s) should first be asked for consent.
+- 2e+) [CLARIFICATION] If a competitor holds multiple citizenship, they may choose which country to represent at their first competition.
```

Comment:
Looking at this more closely, I'm not sure what this accomplishes. Should this be "must"?